data.py-dist

```diff
@@ -50,14 +50,28 @@ VM_IMAGES = {
 }
 
 # FIXME: use URLs and an optional cache?
+#
+# If net-only is set to True only URL source will be possible. By default the
+# parameter is set to False so local source and url, if one is defined
+# in NETINSTALL_URL, are used.
 ISO_IMAGES = {
-#    'xcpng-8.0': "/home/user/iso/xcp-ng-8.0.0.iso",
-#    'xcpng-8.1': "/home/user/iso/xcp-ng-8.1.0-2.iso",
-#    'xcpng-8.2.0': "/home/user/iso/xcp-ng-8.2.0.iso",
-#    'xcpng-8.2.1': "/home/user/iso/xcp-ng-8.2.1.iso",
-#    'xcpng-8.2.1-2023': "/home/user/iso/xcp-ng-8.2.1-20231130.iso",
-#    'xcpng-8.3-beta2': "/home/user/iso/xcp-ng-8.3.0-beta2.iso",
-#    'xcpng-8.3-beta2-net': "/home/user/iso/xcp-ng-8.3.0-beta2-netinstall.iso",
+#    'xcpng-8.0': {'path': "/home/user/iso/xcp-ng-8.0.0.iso"},
+#    'xcpng-8.1': {'path': "/home/user/iso/xcp-ng-8.1.0-2.iso"},
+#    'xcpng-8.2.0': {'path': "/home/user/iso/xcp-ng-8.2.0.iso"},
+#    'xcpng-8.2.1': {'path': "/home/user/iso/xcp-ng-8.2.1.iso"},
+#    'xcpng-8.2.1-2023': {'path': "/home/user/iso/xcp-ng-8.2.1-20231130.iso"},
+#    'xcpng-8.3-beta2': {'path': "/home/user/iso/xcp-ng-8.3.0-beta2.iso"},
+#    'xcpng-8.3-beta2-net': {
+#       'path': "/home/user/iso/xcp-ng-8.3.0-beta2-netinstall.iso",
+#       'net-only': True,
+#    },
+}
+
+# If an URL is set the source type will be either local or url
```

Comment:
Need to be rewrite because in fact source local and netinstall are always generated and the check that configuration is valid is done when remastering the ISO.